tests/cypress/integration/actions_tasks_objects/case_60_autoborder_feature.js

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('Autoborder feature.', () => {
+    const caseId = '60';
+    const createRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 400,
+        firstY: 350,
+        secondX: 500,
+        secondY: 450,
+    };
+
+    const createRectangleShape2PointsSec = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 600,
+        firstY: 350,
+        secondX: 700,
+        secondY: 450,
+    };
+
+    const keyCodeN = 78;
+    let rectangleSvgJsCircleId = [];
+    let rectangleSvgJsCircleIdSecond = [];
+    let polygonSvgJsCircleId = [];
+    let polylineSvgJsCircleId = [];
+
+    function testCollectCxCircleCoord(arrToPush) {
+        cy.get('circle').then((circle) => {
+            for (let i = 0; i < circle.length; i++) {
+                if (circle[i].id.match(/^SvgjsCircle\d+$/)) {
+                    cy.get(`#${circle[i].id}`)
+                        .invoke('attr', 'cx')
+                        .then(($circleCx) => {
+                            arrToPush.push($circleCx);
+                        });
+                }
+            }
+        });
+    }
+
+    function testAutoborderPointsCount(expextedCount) {
+        cy.get('.cvat_canvas_autoborder_point')
+            .should('exist')
+            .and('be.visible')
+            .then(($autoborderPoints) => {
+                expect($autoborderPoints.length).to.be.equal(expextedCount);
+            });
+    }
+
+    function testActivatingShape(x, y, expectedShape) {
+        cy.get('.cvat-canvas-container').trigger('mousemove', x, y);
+        cy.get(expectedShape).should('have.class', 'cvat_canvas_shape_activated');
+    }
+
+    before(() => {
+        cy.openTaskJob(taskName);
+        cy.createRectangle(createRectangleShape2Points);
+        cy.createRectangle(createRectangleShape2PointsSec);
+
+        // Collect the rectagle points coordinates
+        testActivatingShape(450, 400, '#cvat_canvas_shape_1');
+        testCollectCxCircleCoord(rectangleSvgJsCircleId);
+        testActivatingShape(650, 400, '#cvat_canvas_shape_2');
+        testCollectCxCircleCoord(rectangleSvgJsCircleIdSecond);
```

Comment:
Should we test anything in before?

Reply:
Moved to the 1st it(). These functions simply collect parameter values.